webapp/src/components/workspaceSwitcher/workspaceSwitcher.tsx

```diff
@@ -62,9 +62,8 @@ const WorkspaceSwitcher = (props: Props): JSX.Element => {
                             newPath = '/dashboard'
                         } else {
                             newPath = `/workspace/${workspaceId}`
+                            UserSettings.lastWorkspaceId = workspaceId
                         }
```

Comment:
I saw the original bug ticket. You can make this change so that after a CTRL+R you still land on the dashboard.
```suggestion
                            newPath = '/dashboard'
                            UserSettings.lastWorkspaceId = null
                        } else {
                            newPath = `/workspace/${workspaceId}`
                            UserSettings.lastWorkspaceId = workspaceId
                        }
```
I tried it locally and it worked on the desktop app. You can try it out and report if you have the same result.
Thanks

Reply:
It works only when you don't have the last board and view ID in local storage. Can we address this separately to avoid delaying this issue?

Reply:
@harshilsharma63 

Why couldn't we set those to null? Does that cause some other issues?